openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -1197,6 +1203,78 @@ message BidRequest {
     extensions 100 to 9999;
   }
 
+  // OpenRTB 2.6: Structured user agent information, which can be used when a client supports
+  // User-Agent Client Hints. If both device.ua and device.sua are present in the bid request,
+  // device.sua should be considered the more accurate representation of the device attributes.
+  // This is because the device.ua may contain a frozen or reduced user agent string
+  // due to deprecation of user agent strings by browsers.
```

Comment:
Nitpick, but the comment here is almost the same as the comment above. Typically there's the full comment on the `message` definition and then a shorter summary where the field is used.

Reply:
Agree. Please keep this comment and abbreviate the other one to reference this section, in the same way as this:
```
    // An array of Metric object (Section 3.2.5).
    repeated Metric metric = 17;
```

Reply:
The Section reference for Metric in the example above is accurate but the one in line 1038 seems incorrect. 
```
// Location of the device assumed to be the user's current location defined
    // by a Geo object (Section 3.2.12).
    // RECOMMENDED by the OpenRTB specification.
    optional Geo geo = 4;
```
There's a few more incorrect ones, and I don't know which spec they're referring to (checked v2.6, v2.5, v2.0). Ill update the Section references in this PR to match v2.6 to avoid confusion (few others in User message are invalid too - going by OpenRTB v2.6 spec and double-checking against v2.5 for correctness)

Reply:
There is a little discrepancy in the Device object where the `geo` and `sua` fields are now referencing Section numbers from different specs.

Didn't update the Geo field section reference because it is also used by other messages and it causes a discrepancy of Section numbers for the same object. 

Will update the Section references in a different PR.